docs/recipes/ui/add-routing.md

```diff
@@ -0,0 +1,203 @@
+﻿# How do I add routes to a SAFE app without splitting the app into multiple models?
+
+*Written for SAFE template version 4.2.0*
+
+When building larger apps, you probably want different pages to be accessible through different URLs. In this recipe, we show you how to add routes to different pages to an application, including adding a "page not found" page that is displayed when an unknown URL is entered.
+
+## 1. Adding the Feliz router
+
+Install Feliz.Router in the client project
+
+```bash
+dotnet paket add Feliz.Router -p Client -V 3.8
+```
+
+!!! Warning "Feliz.Router versions"
```

Comment:
Didn't know you could do this, it's cool! How did you find out about it?

Reply:
It's in the docs for Admonition https://python-markdown.github.io/extensions/admonition/